File: GA360.DAL.Entities/Entities/Country.cs
using GA360.DAL.Entities.BaseEntities;

namespace GA360.DAL.Entities.Entities;

public class Country : Audit, IModel
{
    public int Id { get; set; }
    public Guid? TenantId { get; set; }
    public string Name { get; set; }
    public string? Code { get; set; }
    public string? Prefix { get; set; }
}

File: ga360.client/src/types/country.ts
export interface Country {
    id: number;
    tenantId: number | null;
    name: string;
    code: string;
    prefix: string;
    isDeleted: boolean;
    deleterUserId: number | null;
    deletionTime: string | null;
    createdBy: string;
    modyfiedBy: string;
    createdAt: string;
    modifiedAt: string;
}
